debian/changelog

```diff
@@ -1,3 +1,7 @@
+  [Paolo Tormene]
+  * Renamed outputs `rates_by_src` to `mean_rates_by_src` and
+    `mean_disagg_bysrc` to `mean_disagg_by_src`
+
```

Comment:
No need to put this in the changelog since the outputs are new in master (i.e. the change does not affect users).

Reply:
Removed